tempalte2.yaml

```diff
@@ -1,3 +1,4 @@
 {
 "awsTokens":"sfgsdegedfgsev234"
+password:dfgkldjfgfrgb54ytesw
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:** password:********************  
[This comment was created by Aqua Pipeline]